hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreFileInfo.java

```diff
@@ -107,6 +107,8 @@
 
   private RegionCoprocessorHost coprocessorHost;
 
+  private boolean prefetchOnOpen = false;
```

Comment:
The trick is here. By default it will be false and only for region open and the bulk load  you will make it true. Good. LGTM.